modules/clientmanagements/client/views/create-clientmanagement.client.view.html

```diff
@@ -53,7 +53,7 @@
         <br>
         <div class="form-group">
           <div class="form-group pull-right">
-          <button type="submit" class="btn btn-default" href="/client/views/view-clientmanagement.client.view.html">{{vm.clientmanagement._id ? 'Update' : 'Create'}}</button>
```

Comment:
This is how it determines if it should say update or create, why delete it?

Reply:
Since it is a form, you can just have a submit button.